pages/room/index.js

```diff
@@ -18,8 +19,26 @@ const CreateRoom = () => {
     initSession();
   }, []);
 
-  function create() {
+  async function create() {
     const id = uuid();
+    const currentUser = await axios.get('/api/get-user', {
+      params: {
+        name: 'Eden Chan',
+        email: 'edenchan717@gmail.com',
+      },
+    });
+
+    const currentUserId = currentUser.data.user._id;
+
+    const result = await axios.post('/api/spaces/create-new-space', {
+      name: 'test name',
+      description: 'test descritpion',
+      isActive: true,
+      participants: [currentUserId],
+      spaceId: id,
+    });
+    console.log(result);
+    console.log(currentUserId);
```

Comment:
never put console logs into production. Only `console.debug` is acceptable. Even then, we only want to console data if you think it's absolutely necessary.